taxi/urls.py

```diff
@@ -46,6 +50,14 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create", DriverCreateView.as_view(), name="driver-create"),
+    path("drivers/delete", DriverDeleteView.as_view(), name="driver-delete"),
```

Comment:
The path for 'drivers/delete' is missing a trailing slash. It should be 'drivers/delete/' to maintain consistency with other URL patterns.